gce/prepull.yaml

```diff
@@ -13,7 +30,7 @@ spec:
     spec:
       nodeSelector:
         kubernetes.io/os: windows
-      initContainers:
+      containers:
```

Comment:
do you also need a RestartPolicy?

Reply:
I tried but it turns out that DaemonSets don't support RestartPolicy - only pods :'(

Reply:
then how about making them 'sleep'? doe that work?

Reply:
Ok, I think I got something that will work for an infinite loop:

```
command: ['cmd.exe', '/c', 'ping -n 1800 127.0.0.1 >NUL']
```

I tried for half an hour to get the built-in cmd.exe command for sleep, `TIMEOUT`, to work but apparently it fails in containers and scripts: https://www.ibm.com/support/pages/timeout-command-run-batch-job-exits-immediately-and-returns-error-input-redirection-not-supported-exiting-process-immediately. Ugh.

I'll update this and re-run everything and will update the PR.